assets/wizards/payment/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Payment
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { Component, render, Fragment, createElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Material UI dependencies.
+ */
+import HeaderIcon from '@material-ui/icons/Dns';
+
+/**
+ * Internal dependencies.
+ */
+import { withWizard } from '../../components/src';
+import Router from '../../components/src/proxied-imports/router';
+import { PaymentMethod } from './views';
+
+const { HashRouter, Redirect, Route, Switch } = Router;
+
+class PaymentWizard extends Component {
+	state = {
+		customer: {},
+	};
+	componentDidMount = props => {
+		this.retrieve();
+	};
+	retrieve = () => {
+		const { setError, wizardApiFetch } = this.props;
+		return wizardApiFetch( { path: '/newspack/v1/wizard/newspack-payment-wizard/' } )
+			.then( ( { customer, subscription } ) => this.setState( { customer, subscription } ) )
+			.catch( error => setError( error ) );
+	};
+	checkout = () => {
+		const { setError, wizardApiFetch } = this.props;
+		return wizardApiFetch( { path: '/newspack/v1/wizard/newspack-payment-wizard/checkout' } )
+			.then( data => {
+				const stripe = Stripe( data.stripe_publishable_key );
```

Comment:
In https://github.com/Automattic/newspack-plugin/pull/442/commits/77615150009528acd95e363d5cb2fd898408f0d6 I switched to the NPM Stripe library, instead of simply adding a script tag. This effectively addresses the point about `Stripe` as a global. Other linting errors resolved in https://github.com/Automattic/newspack-plugin/pull/442/commits/204ec178c4b9962a82d173a3137d676802d0a4ae